app/views/unattended/.gitignore

```diff
@@ -0,0 +1 @@
+Gemfile
```

Comment:
What's the background of this?

Reply:
I see, personally I would add the full path in the main .gitignore then

Reply:
I'll just get rid of the gitignore entirely